### ISSUE TITLE
Growthcraft Rice: Bonemeal Sometimes Won't Cause Growth

### ISSUE THREAD

**Issue by Nakuta** [GrowthcraftCE/Growthcraft-1.12#116]
Version 4.0.4.400
-Rice can now be bone-mealed.
Sometimes bonemeal doesn't appear to work. I get the green particle affect and consume bonemeal, but I can hold down the button for over a minute and nothing will grow. I can't replicate this 100% of the time, but most often happens when I create a rice paddy, immediately plant rice and then immediately bonemeal the just-planted stage 0 rice.

**Comment by Alatyami** [comment]
Will need to confirm, but bonemeal is generally set to work 25% of the time. We'll look into as a balancing item.

**Comment by Nakuta** [comment]
Retested this for build growthcraft-1.12.2-4.0.4.400.DEV.52b3e0d and confirming that it's still a problem.

I did a more in-depth test and the bug only occurs when you create a rice paddy and then immediately plant rice on it. This becomes bugged and will accept bonemeal but won't grow from it; my tests included holding down the button in creative to repeatedly bonemeal the stage 0 rice for up to 5 minutes. Once the rice plant naturally progresses to stage 1, the issue resolves itself and the rice plant will accept and respond to bonemeal as expected.

When creating a rice paddy and then waiting 30+ seconds before planting, I did not encounter this bug, so it seems to be something that involves recently-created rice paddies.
